src/routes/api/comments.js

```diff
@@ -0,0 +1,22 @@
+import { Router } from 'express';
+import CommentController from '../../controllers/CommentController';
+import Validation from '../../middlewares/validateComment';
+import checkArticle from '../../middlewares/checkArticle';
+import checkComment from '../../middlewares/checkComment';
+import errorHandler from '../../middlewares/errorHandler';
+
+const router = Router();
+
+// router.post('/:articleId/comments', Validation, checkArticle, CommentController.create);
```

Comment:
@MBenedicte delete this commented out code

Reply:
YEAH